README.md

```diff
@@ -49,6 +49,14 @@ Then you can run `rake db:migrate`, `rspec`, whatever.  Or you can run those dir
 $> docker compose exec api rake db:migrate
 ```
 
+#### Updating rails credentials
+```bash
+EDITOR=vim bin/rails credentials:edit --environment development
+EDITOR=vim bin/rails credentials:edit --environment test
+EDITOR=vim bin/rails credentials:edit --environment staging
+EDITOR=vim bin/rails credentials:edit --environment production
```

Comment:
haha we won't even need this anymore since i ripped out the credentials, i can remove this! you can pass in whatever EDITOR you want ie vscode would be `EDITOR=code`